lib/components/map/default-map.tsx

```diff
@@ -269,31 +279,44 @@ class DefaultMap extends Component {
     return (
       <MapContainer>
         <BaseMap
-          baseLayers={baseLayersWithNames}
+          // Only 1 base layer is supported at the moment
+          baseLayer={baseLayersWithNames?.[0]?.url}
           center={center}
+          mapLibreProps={{ reuseMaps: true }}
           maxZoom={maxZoom}
-          onClick={this.onMapClick}
+          onContextMenu={this.onMapClick}
           onPopupClosed={this.onPopupClosed}
+          onViewportChanged={() => {
+            // This hack is needed to get the zoom functionality to work
+            // Redux only fires the render method if the prop changes,
+            // so it must be set to null so it can be "re-set" to an
+            // actual value.
+            if (mapConfig.initLat || mapConfig.initLon) {
+              setMapCenter({ lat: null, lon: null })
+            }
+          }}
```

Comment:
Move to a class method.

Reply:
Disregard the portion of my comment above regarding setting up a new redux state. It doesn't look like we are using the current map bounds anywhere except this hack so this event handler should be deleted. See my general comment regarding setting the map center dynamically.